mathesar/serializers.py

```diff
@@ -59,6 +70,15 @@ def to_representation(self, instance):
         return instance._asdict()
 
 
+class DatabaseSerializer(serializers.ModelSerializer):
```

Comment:
The entire serializer should be read only. Databases should not be able to be edited or updated anywhere but in the settings.